src/main/java/string/Exercise04.java

```diff
@@ -0,0 +1,26 @@
+package string;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+public class Exercise04 {
+    public boolean checkRepeatOneString(String str) {
+        if (str.length() < 2) return false;
+
+        Set<String> set = new HashSet<String>();
+        for (String w : str.split(str.charAt(0) + "+")) {
+            set.add(w);
+        }
```

Comment:
Không chắc ý tưởng của em lắm nhưng đoạn code này có thể viết thế này
```suggestion
        Set<String> set = new HashSet<>(Arrays.asList(str.split(str.charAt(0) + "+")));
```

Reply:
Done